### PR TITLE
fix: use version from environment if already downloaded one

### LINKED ISSUE #383: Use upstream binary provided by `EC_VERSION`
When the tool is run without an argument it downloads the latest binary.
If I after that run the tool again with `EC_VERSION=2.3.0` it auto picks the already downloaded binary instead of downloading the specified version.

### PR DESCRIPTION
**Pull request by mstruebing**
Closes #383

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -29,9 +29,10 @@ async function execute() {
 
 async function isReady() {
   try {
-    await fs.stat(binPath)
+    await fs.stat(COMBINED_PATH)
     return true
   } catch {
+    fs.mkdir(COMBINED_PATH, { recursive: true })
```
Comment (septs) [inline src/index.ts:35]: why not `await`?

**Review by Divlo — APPROVED** [review]
Looks Good To Me once https://github.com/editorconfig-checker/editorconfig-checker.javascript/pull/384#discussion_r870962103 is resolved (it should be using `await`).

Any chance, we can release v5.0.0 (including this PR) on `npm`?
Currently, the latest version published (v4.0.2), is using `editorconfig-checker` v2.3.5 whereas v2.6.0 is already available (quite a few versions behind). @mstruebing @septs 

Thanks! :smile:

**Comment by Divlo** [comment]
cc @mstruebing @septs

**Review by septs — APPROVED** [review]

File: src/index.ts (hunk 1 above)
Comment (mstruebing) [reply src/index.ts]: Added `await`.

**Pull request merged**